lib/tandem/tandemSimulator.js

```diff
@@ -176,8 +170,14 @@ exports.make = function(config){
     smbg: function(event) {
       simpleSimulate(event);
     },
-    suspend: function(event) {
-      simpleSimulate(event);
+    suspendResume: function(event) {
```

Comment:
I'm not really a fan of collapsing the suspend & resume logic in the same place. Sell it to me? Your advertisement should include a clearer function name than `suspendResume` which sounds like you're suspending a resume event (to me).